src/System.Data.SqlClient/tests/FunctionalTests/AADAccessTokenTest.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Data.SqlClient.Tests
+{
+    public class AADAccessTokenTest
+    {
+        SqlConnectionStringBuilder _builder;
+
+        [Fact]
+        public void InvalidCombinationOfAccessToken()
```

Comment:
This test can be parameterized by passing different connection strings, perhaps by using [Theory] and [InlineData] attribute.